eng/Generate.ps1

```diff
@@ -17,13 +17,14 @@ function Invoke-Block([scriptblock]$cmd) {
     }
 }
 
-function Invoke-AutoRest($debugFlags, $testConfiguration, $outputFolder, $inputFile, $name, $namespace, $repoRoot) {
+function Invoke-AutoRest($debugFlags, $requireFile, $outputFolder, $inputFile, $title, $namespace, $repoRoot) {
```

Comment:
Can you change this method to take just an array of arguments or string with all arguments?

Reply:
Not all calls to this function currently use all arguments. These just become named arguments. If I remove the named arguments, all arguments are accessible by $args, but become position-bound (meaning, you have to know which index into $args represents which data). Even with named arguments, you can still access all arguments via $args.

However, I can try to clean this up. I just didn't want to spent a lot of time on it.

Reply:
Done. Made Invoke-AutoRest take in a single $autoRestArguments parameter.